lib/streamlit/elements/arrow_vega_lite.py

```diff
@@ -15,9 +15,9 @@
 """A Python wrapper around Vega-Lite."""
```

Comment:
I think we can also add the `theme` parameter to the `_arrow_vega_lite_chart` method.